Helpers/SecurityActionHelpers.cs

```diff
@@ -0,0 +1,25 @@
+namespace Cliptok.Helpers
+{
+    public class SecurityActionHelpers
+    {
+        public static async Task<JToken> GetCurrentSecurityActions(ulong guildId)
+        {
+            using HttpRequestMessage getActionsRequest = new(HttpMethod.Get, $"https://discord.com/api/v9/guilds/{guildId}");
+            getActionsRequest.Headers.Authorization = new AuthenticationHeaderValue("Bot", Environment.GetEnvironmentVariable("CLIPTOK_TOKEN") ?? Program.cfgjson.Core.Token);
+            
+            var getActionsResponse = await Program.httpClient.SendAsync(getActionsRequest);
+            return ((JObject)JsonConvert.DeserializeObject(await getActionsResponse.Content.ReadAsStringAsync()))["incidents_data"];
+        }
+
+        public static async Task<HttpResponseMessage> SetCurrentSecurityActions(ulong guildId, string newSecurityActions)
+        {
+            // create & send request
+            
+            using HttpRequestMessage setActionsRequest = new(HttpMethod.Put, $"https://discord.com/api/v9/guilds/{guildId}/incident-actions");
```

Comment:
Lets make this dynamic so when we do upgrade API version they're all done at the same time.
```suggestion
            using HttpRequestMessage setActionsRequest = new(HttpMethod.Put, $"https://discord.com/api/v{Program.discord.GatewayVersion}/guilds/{guildId}/incident-actions");
```